dask/bag/tests/test_bag.py

```diff
@@ -1473,3 +1473,12 @@ def f_drop(o):
         b.compute(scheduler="sync")
     finally:
         gc.enable()
+
+
+def test_bagged_array_delayed():
+    import dask.array as da
```

Comment:
Please defend this with `pytest.importorskip` (see other tests in this file for examples)